doc/manual/src/language/constructs.md

```diff
@@ -217,18 +217,20 @@ three kinds of patterns:
     
     > **Warning**
     > 
-    > The `args@` expression is bound to the argument passed to the
-    > function which means that attributes with defaults that aren't
-    > explicitly specified in the function call won't cause an
-    > evaluation error, but won't exist in `args`.
+    > The `args@` expression is bound to the argument that is passed to the
+    > function. This does not take default values into account.
```

Comment:
```suggestion
    > `args @` binds `args` to *exactly* argument that is passed to the function.
    > In particular, `args` does *not* include any default values for attributes specified with a `?` in the function's set pattern.
```

Reply:
```suggestion
    > `args @` binds `args` to the attribute set that was *originally* passed to the function.
    > In particular, `args` does *not* include any default values for names specified with a `?` in the function's set pattern.
```

argument -> attribute set [that was passed]: it being an attrset differentiates it from the deconstructed names.

default values for attributes -> default values for names: This was not the right mental model. The default only applies to the binding, not to the attribute.
I'd say _bindings_ but _names_ is more accessible, and more differentiated from the earlier use of _binds_.